### PR TITLE
[#69] Cleanup more deps. Remove Serverless.

### LINKED ISSUE #69: Update deps: part 2
Revenge of #67 

`grpcio` was obsolete because `azure-functions-worker` was obsolete. Trying the [official MS guide](https://learn.microsoft.com/en-us/azure/azure-functions/create-first-function-cli-python?tabs=linux%2Cbash%2Cazure-cli%2Cbrowser#create-a-local-function), the requirements only include `azure-functions`, and nothing else.

See if the deployment works without `azure-functions-worker`.

### PR DESCRIPTION
**Pull request by danuker**
Fixes #69 

Cleaned up some more. We don't need Serverless nor Node anymore either; updated `README.rst`.

Deployed the new version. Let's see if it works (and requests your review).

reviewers: @adiroiban

### CODE REVIEW

**Comment by danuker** [comment]
needs-review

File: README.rst [hunk 1]
```diff
@@ -21,19 +21,10 @@ Virtual environment
 
 To create a working virtual environment::
 
-    virtualenv -p 3.8 venv  # Last Python version supported by serverless-azure-functions
+    virtualenv -p 3.11 venv
```
Comment (adiroiban) [inline README.rst:24]: why not got with 3.12 ? Just asking

**Review by adiroiban — APPROVED** [review]
Thanks for the cleanup

File: README.rst (hunk 1 above)
Comment (danuker) [reply README.rst:24]: 3.11 is the latest supported for now: https://learn.microsoft.com/en-us/azure/azure-functions/supported-languages?tabs=isolated-process%2Cv4&pivots=programming-language-python#languages-by-runtime-version

**Pull request merged**